### ISSUE TITLE
Agents registration manual typo

### ISSUE THREAD

**Issue by oldmerkum** [wazuh/wazuh-documentation#689]
https://github.com/wazuh/wazuh-documentation/blob/3.7/source/user-manual/agents/command-line/register.rst

This section states to update the parameter <server-ip> in the client config file. The parameter should be <address> within the <server> tag.

**Comment by JuanjiJG** [comment]
Hi @oldmerkum,

Thank you for reporting this. Indeed, the documentation is outdated. In Wazuh 3.7.0, that config block looks like the following on an agent:

```
<client>
  <server>
    <address>MANAGER_IP</address>
    <port>1514</port>
    <protocol>udp</protocol>
  </server>
  ...
</client>
```

You have to update the `<address>` field with the manager's IP address.

We'll update the documentation to reflect this changes. Apologies for the inconvenience.

Regards,
Juanjo

**Comment by JuanjiJG** [comment]
Done here: #758.

**Comment by JuanjiJG** [comment]
I've merged #758, so I'll proceed to close this issue.

Regards,
Juanjo
